src/Components/ProvisioningWizard/steps/Pubkeys/NewKeyForm.js

```diff
@@ -109,15 +112,13 @@ const NewSSHKeyForm = ({ setStepValidated }) => {
         />
       </FormGroup>
       <FormGroup
```

Comment:
Could we add the `?` when there are some unsuported keys?
When I select a `ed25519` key, I just get "is invalid or unsupported", but as a user, I might not know what does it mean.
If we would have the popup help when the provider is Azure (I guess this is ok to do statically) saying "Azure does not support ed25519 keys" and link to Azure statement about this, for example: https://learn.microsoft.com/en-us/troubleshoot/azure/virtual-machines/ed25519-ssh-keys I guess users would appreciate this (as it was very unexpected information for me, so I guess it will surprise many users and we don't want to get the blame if it's not our fault ;) )